artacle-swagger.yaml

```diff
@@ -314,6 +314,281 @@ paths:
+        - name: sortBy
+          in: query
+          required: false 
+          description: Sort result by one of criteria
+          schema:
+            type: string
+            enum:
+              - valuationAllTime
+              - valuationRecent
+              - floor
+              - totalSalesVolume
+              - totalAmountOfSales
+            default: valuationAllTime
+            example: valuationAllTime
+        - name: sortingDirection
+          in: query
+          required: false
+          description: "Sorting direction: ascending, descending"
+          schema:
+            type: string
+            enum:
+              - asc
+              - desc
+            default: desc
+            example: desc
+        - name: offset
+          in: query
+          required: false
+          description: Results pagination
+          schema:
+            type: integer
+            default: 0
+            example: 0
+        - name: limit
+          in: query
+          required: false
+          description: Limit elements to fetch
+          schema:
+            type: integer
+            example: 50
+            default: 50
+            maximum: 1000
+      responses:
+        '200':
+          description: Success, returns array of collection traits, its valuation and information.
+          content:
+            application/json:
+              schema:
+                type: object
+                required:
+                  - data
+                properties:
+                  data:
+                    type: array
+                    description: Array of Traits info
+                    items:
+                      $ref: '#/components/schemas/CollectionTraits'
+        '401':
+          $ref: "#/components/responses/UnauthorizedError"
+        '403':
+          $ref: "#/components/responses/ForbiddenError"
+        '404':
+          $ref: "#/components/responses/NotFoundError"
+        '429':
+          $ref: "#/components/responses/TooManyRequestsError"
+        '500':
+          $ref: "#/components/responses/InternalServerError"
+
+#  Collection Trait
+  /collections/{cid}/trait/v1:
+    get:
+      tags:
+        - Traits
+      summary: Get particular trait information and valuation within a specific collection. All prices and volumes by default are provided in the native collection currency (i.e. ETH for Ethereum)
+      description: Returns particular collection trait information
+      parameters:
+        - name: cid
+          in: path
+          required: true
+          description: Artacle collection ID as returned by the /collections/ endpoint
+          schema:
+            type: integer
+            example: 78
+        - name: apiKey
+          in: query
+          required: true
+          description: API access key
+          schema:
+            type: string
+        - name: traitName
+          in: query
+          required: true
+          description: Query only traits with certain name
```

Comment:
It is never 'traitS', it is always 1 trait, the description confuses. Suggesting: "Name of the trait", "Value of the trait"